benchmarks/helics/EchoHubFederate.hpp

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright (c) 2017-2020,
+Battelle Memorial Institute; Lawrence Livermore National Security, LLC; Alliance for Sustainable Energy, LLC.  See
+the top-level NOTICE for additional details. All rights reserved.
+SPDX-License-Identifier: BSD-3-Clause
+*/
+
+#include "BenchmarkFederate.hpp"
+#include "helics/application_api/Inputs.hpp"
+#include "helics/application_api/Publications.hpp"
+#include "helics/application_api/Subscriptions.hpp"
+
+using helics::operator"" _t;
+/** class implementing the hub for an echo test*/
+class EchoHub: public BenchmarkFederate {
+  private:
+    std::vector<helics::Publication> pubs;
+    std::vector<helics::Input> subs;
+    int cnt = 10;
+
+  public:
+    EchoHub(): BenchmarkFederate("echo hub benchmark federate") {}
+
+    void setupArgumentParsing() override
+    {
+        // default final time for this benchmark
+        finalTime = helics::Time(100, time_units::ms);
+
+        app->add_option("--num_leafs", cnt, "the number of echoleaf federates to expect")
```

Comment:
I would add a true to the call, so the 10 (default) shows up.  
Why is this required?

Reply:
I marked the parameters that were previously passed in using the `initialize` calls as required. This one probably doesn't need to be required since the default is okay.